tests/test_flake8_trio.py

```diff
@@ -465,12 +467,42 @@ async def foo():
     sync_fns.takes_a_long_time()
 """
     )
+    return (
+        "./example.py:5:5: TRIO200: User-configured blocking sync call sync_fns.* "
+        "in async function, consider replacing with the_async_equivalent.\n"
+    )
+
+
+def test_200_from_config_flake8_internals(
+    tmp_path: Path, capsys: pytest.CaptureFixture[str]
+):
+    # abuse flake8 internals to avoid having to use subprocess
+    # which breaks breakpoints and hinders debugging
+    # TODO: fixture (?) to change working directory
+
+    err_msg = _test_trio200_from_config_common(tmp_path)
+    # replace ./ with tmp_path/
+    err_msg = str(tmp_path) + err_msg[1:]
+
+    from flake8.main.cli import main
+
+    main(
+        argv=[
+            str(tmp_path / "example.py"),
+            "--append-config",
+            str(tmp_path / ".flake8"),
+        ]
+    )
+    out, err = capsys.readouterr()
+    assert not err
+    assert err_msg == out
+
+
+def test_200_from_config_subprocess(tmp_path: Path):
+    err_msg = _test_trio200_from_config_common(tmp_path)
     res = subprocess.run(["flake8"], cwd=tmp_path, capture_output=True)
     assert not res.stderr
-    assert res.stdout == (
-        b"./example.py:5:5: TRIO200: User-configured blocking sync call sync_fns.* "
-        b"in async function, consider replacing with the_async_equivalent.\n"
-    )
+    assert res.stdout == bytes(err_msg, "ascii")
```

Comment:
```suggestion
    assert res.stdout == err_msg.encode("ascii")
```

Semantically identical in this case, but I prefer `str.encode()` for clarity as bytes() is also callable on e.g. a list of ints.